source/_posts/2016-09-16-how-to-write-open-source-in-php-2-enjoy-skeletons.md

```diff
@@ -104,7 +104,7 @@ Just to remind you, the end user is interested in:
 
 *Meaning*
 
-- here are all files that are ignored by composer
+- here are all files that are ignored by composer (with `export-ignore` flag)
```

Comment:
This confused me. Compose with flag `--export-ignore`? Or files with flag?
Could you rewrite?